ar_module2_8.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+var myDate = (function iffe() {
+  /* eslint-disable no-console */
+  /* eslint-disable no-unused-vars */
+  var privateDate = new Date(); // make this available to both get and set
+  var dd = privateDate.getDate();
+  var mm = privateDate.getMonth() + 1; //January is 0!
+  var yyyy = privateDate.getFullYear();
+
+  if (dd < 10) {
+    dd = '0' + dd;
+  }
+  if (mm < 10) {
+    mm = '0' + mm;
+  }
+
+  var checkDate = mm + dd + yyyy;
+  var formatDate = mm + '/' + dd + '/' + yyyy;
+
+  function setDate(newDate) {  // make a function for milli
+
+    if (newDate == undefined) {
+      var defaultDate = privateDate;
```

Comment:
Since defaultDate is only scoped to this function, this statement doesn't actually accomplish anything. If the user doesn't pass in an argument to this method, you should set the internally stored date to "now". This means simply setting it to a new instance of Date without passing in any arguments.